ImprovedAnvil/lang/English/setup.tra

```diff
@@ -7615,4 +7616,466 @@ Armor Class: 4 (6 vs. piercing and missile)
 Requires:
  6 Strength
 
-Weight: 28~
\ No newline at end of file
+Weight: 28~
+
+@12080 = ~Weaken Undead~
+
+@12081 = ~Weaken Undead
+(Abjuration)
+
+Level: 5
+Range: 0 
+Duration: 6 rounds
+Casting Time: 5 
+Area of Effect: The caster
+Saving Throw: None
+
+This powerful spell creates waves of anti-negative energy that sweep outward from the caster. These waves disrupt any undead that attempt to attack the caster, reducing the physical damage they deal by 10% and decreasing their chance for critical hit by 5% for 7 seconds. There is one wave per round for the duration of the spell, which renews the effects, but does not stack with itself. All undead are affected with no Saving Throw, this spell bypasses Magic Resistance.~
```

Comment:
anti-negative - positive?

Reply:
yeah, this probably makes sense, as its positive energy that affects the undead.